.github/workflows/run-tests.yml

```
@@ -0,0 +1,18 @@
+name: Continuous Integration
+on: 
+    push:
+        paths-ignore:
+            - 'README.md'
+    pull_request:
+        # Sequence of patterns matched against refs/heads
+        branches:    
+            - main  
+
+jobs:
+    build:
+        name: Build and test
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
+        - run: docker build --target test --tag todo-app:test .
+        - run: docker run todo-app:test
```

Your tests happen to work with an empty environment, so this isn't necessary currently, but you will likely need it for the next module.

```suggestion
        - run: docker run --env-file .env.test todo-app:test
```